src/events.ts

```diff
@@ -12,15 +14,28 @@ export class EventArgs {
     }
 }
 
-export abstract class EventEmitter {
+export class EventEmitter {
```

Comment:
We now create an instance of this within Container to be used for sending static events. The ipc/MessageBus is provided on the constuctor which causes that emitter to subscribe for custom static event messages sent on the bus for forwarding.